source/connect/in-use-encryption.txt

```diff
@@ -0,0 +1,233 @@
+.. _in-use-encryption-tab:
+
+=================
+In-Use Encryption 
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
```

Comment:
We need a [call out to let people know that QE ](https://www.mongodb.com/docs/upcoming/core/queryable-encryption/) is still in technical preview mode. This is done in a [banner on the server docs](https://github.com/10gen/docs-mongodb-internal/blob/master/snooty.toml#L277-L292): 

The verbiage is:

```
Queryable Encryption is in Technical Preview and available for \
evaluation purposes. Technical Preview is not recommended for \
production deployments as breaking changes may be introduced.
```

Reply:
Done.